portal/client/public/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
```

Comment:
Lets update this file with proper names and use the icons from `src/public/icon`, look in the `qTrack` project for same, if did not copied earlier